contracts/LobstersNft.sol

```diff
@@ -52,7 +52,7 @@ contract LobstersNft is ILobstersNft, LobstersNames, ERC721, VRFConsumerBase {
     }
   }
 
-  function mint(address _to) public override onlyMinter returns (uint256 id) {
```

Comment:
wondering if there's a clear reason to remove the returned ID? personally have found it useful in some cases

Reply:
The current imp doesn't return an ID.

Reply:
Also the EVM doesn't allow you to change a state and return a value, so there's no reason in returning a result.